Makefile

```diff
@@ -119,7 +119,7 @@ test-coverage: test	  ## Run automated tests and create coverage report
 lint:              		  ## Run code linter to check code style, check if formatter would make changes and check if dependency pins need to be updated
 	($(VENV_RUN); python -m ruff check --output-format=full . && python -m ruff format --check .)
 	$(VENV_RUN); pre-commit run check-pinned-deps-for-needed-upgrade --files pyproject.toml # run pre-commit hook manually here to ensure that this check runs in CI as well
-
+	$(VENV_RUN); python -c 'import localstack.spec, json; print(json.dumps(localstack.spec.OPENAPI))' | openapi-spec-validator -
```

Comment:
Great to see that the validity of the spec is checked on every commit. In order to make sure that a developer gets early feedback when breaking the spec, I think it would make sense to also add the precommit hook (by adding a config to `.pre-commit-config`).
There actually is an official pre-commit hook for the `openapi-spec-validator`: https://github.com/python-openapi/openapi-spec-validator/blob/master/.pre-commit-hooks.yaml